ipaclient/install/client.py

```diff
@@ -126,8 +126,11 @@ def inner(options, tdict):
             pass
         try:
             os.remove(krb_name + ".ipabkp")
```

Comment:
This try/except can be replaced by remove_file().

Reply:
I should definitely consider getting new glasses... Not able to see this method that was like 2 lines below...
Thanks for tip, suggestion implemented.